packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -44,23 +44,27 @@ type TDashboard = {
     onEntered: () => void;
     onOkButtonClick: () => void;
     setActiveTab: (active_tab: number) => void;
-    setBotBuilderTokenCheck: (param: string | number) => void;
     setBotBuilderTourState: (param: boolean) => void;
-    setOnBoardingTokenCheck: (param: string | number) => void;
     setOnBoardTourRunState: (param: boolean) => void;
+    loadDataStrategy: () => void;
+    is_mobile: boolean;
```

Comment:
can we use `isMobile` method from the `@deriv/shared` package?
I remember previously seniors has also suggested the same.

Reply:
@sandeep-deriv done ✅ 
I see that we have the same dependency in many components, which is very bad for striving to make a micro frontend architecture
so, I've created a Redmine card for it ([link](https://redmine.deriv.cloud/issues/81936#Replace_all_variables_is_mobile_from_UI_store_from_the_core_package_to_isMobile_from_'@deriv/shared'_to_avoid_unnecessary_dependency_between_packages_in_bot-web-ui_package))